service/src/test/java/bio/terra/profile/service/profile/ProfileServiceUnitTest.java

```diff
@@ -87,14 +87,14 @@ public void before() {
   }
 
   @Test
-  public void createProfile() {
+  void createProfile() {
     var jobBuilder = mock(JobBuilder.class);
 
     when(jobService.newJob()).thenReturn(jobBuilder);
     when(jobBuilder.description(anyString())).thenReturn(jobBuilder);
-    when(jobBuilder.flightClass(eq(CreateProfileFlight.class))).thenReturn(jobBuilder);
```

Comment:
Why the removal of the argument matchers? 

The direct argument version only works sometimes, but the matchers work in all usages (spy vs mock, when you need other argument matchers, when you need to do the `doReturn()...` construction instead of `when()...`, etc)

I'm not concerned about the change itself, just that the existing pattern would be considered a code smell.

Reply:
https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&id=DataBiosphere_terra-billing-profile-manager&open=AYLbaQuOIfTdJA_2kteU